modules/logging/cores.go

```diff
@@ -0,0 +1,35 @@
+package logging
+
+import (
+	"github.com/caddyserver/caddy/v2"
+	"github.com/caddyserver/caddy/v2/caddyconfig/caddyfile"
+	"go.uber.org/zap/zapcore"
```

Comment:
I _think_ the linter wants it in this order

```suggestion
	"go.uber.org/zap/zapcore"

	"github.com/caddyserver/caddy/v2"
	"github.com/caddyserver/caddy/v2/caddyconfig/caddyfile"
```